### PR TITLE
[Docs] Alternative download bundle for older Kube/OCP 3.11

### PR DESCRIPTION
**Pull request by k-wall**
### Type of change

- Documentation

### Description

For this release there will be two bundles, one suitable for < Kube 1.16 (and OpenShift 3.11) and another suitable for newer Kubernetes and all OpenShift 4 versions. The documentation needs to reflect this.

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: documentation/common/proc-download-procedure.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ ifdef::Downloading[]
 endif::Downloading[]
 
 ifndef::Downloading[]
-* Download and extract the `{ZipInstallFile}` file from the {ZipDownload}.
+* Download and extract the `{ZipInstallFile}` file from the {ZipDownload}.
```
Comment (k-wall) [inline documentation/common/proc-download-procedure.adoc:17]: So for this release there will be two bundles, one suitable for < Kube 1.16 (and OpenShift 3.11) and another suitable for newer Kubernetes and all OpenShift 4 versions.  How best shall this be presented?   Feel free to commit to this PR or I'll make a suggestion tomorrow.  @hmaclean73

File: documentation/common/proc-download-procedure.adoc [hunk 2]
```diff
@@ -10,11 +10,21 @@
 
 .Procedure
 ifdef::Downloading[]
-* Download one of the releases from the link:https://github.com/EnMasseProject/enmasse/releases[GitHub repository] and unpack it.
+
+
+* Download one of the releases from the link:https://github.com/EnMasseProject/enmasse/releases[GitHub repository]. There are two versions bundles of the bundle provided: `enmasse-<version>.tgz` and `enmasse-prekube1_16-<version>.tgz`.
```
Comment (hmaclean73) [inline documentation/common/proc-download-procedure.adoc:15]:
```suggestion
* Download one of the releases from the link:https://github.com/EnMasseProject/enmasse/releases[GitHub repository]. There are two versions of the bundle provided: `enmasse-<version>.tgz` and `enmasse-prekube1_16-<version>.tgz`.
```

File: documentation/common/proc-download-procedure.adoc [hunk 3]
```diff
@@ -10,11 +10,21 @@
...
+ifeval::["{cmdcli}" == "oc"]
+On {KubePlatform} 3.11 and before choose `enmasse-prekube1_16-<version>.tgz`
```
Comment (hmaclean73) [inline documentation/common/proc-download-procedure.adoc:17]:
```suggestion
On {KubePlatform} 3.11 and before, choose `enmasse-prekube1_16-<version>.tgz`.
```

File: documentation/common/proc-download-procedure.adoc [hunk 4]
```diff
@@ -10,11 +10,21 @@
...
+endif::[]
+ifeval::["{cmdcli}" == "kubectl"]
+On {KubePlatform} 1.15 and below choose `enmasse-prekube1_16-<version>.tgz`
```
Comment (hmaclean73) [inline documentation/common/proc-download-procedure.adoc:20]:
```suggestion
On {KubePlatform} 1.15 and below, choose `enmasse-prekube1_16-<version>.tgz`.
```

File: documentation/common/proc-download-procedure.adoc [hunk 5]
```diff
@@ -10,11 +10,21 @@
...
+endif::[]
+otherwise take `enmasse-<version>.tgz`.
```
Comment (hmaclean73) [inline documentation/common/proc-download-procedure.adoc:22]:
```suggestion
Otherwise, choose `enmasse-<version>.tgz`.
```

**Review by hmaclean73 — CHANGES_REQUESTED** [review]
Just a few changes

**Review by hmaclean73 — APPROVED** [review]
LGTM

**Pull request merged**